api/r/cellxgene.census/vignettes/comp_bio_census_info.Rmd

```diff
@@ -0,0 +1,243 @@
+---
+title: "Learning about the CZ CELLxGENE Census"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Learning about the CZ CELLxGENE Census}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+options(width = 88)
+```
+
+This notebook showcases the Census contents and how to obtain high-level information about it. It covers the organization of data within the Census, what cell and gene metadata are available, and it provides simple demonstrations to summarize cell counts across cell metadata.
+
+**Contents**
+
+- Opening the census
+- Census organization
+- Cell metadata
+- Gene metadata
+- Census summary content tables
+- Understanding Census contents beyond the summary tables
+
+## Opening the Census
+
+The `cellxgene.census` R package  contains a convenient `open_soma()` API to open any version of the Census (the latest by default).
```

Comment:
I believe we changed this to `stable` by default, at least in python. Has this been done in R as well @atolopko-czi ? If not, we should probably uniform the behavior and update this vignette.

Reply:
Nope, good call! Added https://github.com/chanzuckerberg/cellxgene-census/issues/482

Reply:
Thanks Andrew! In this case, I will resolve this conversation as we should tackle that in a separate PR. I added a note in the ticket to update the vignette.

Reply:
Agree- I'll take care of #482 once the `stable` key exists in the `release.json`!